src/community/resources/index.md.njk

```diff
@@ -0,0 +1,103 @@
+---
+title: Resources
+description: Useful resources and community projects
+section: Community
+layout: layout-pane.njk
+order: 1
+status: Experimental
+---
+
+A curated list of community developed resources that can help you use the GOV.UK Design System in your chosen design tool or development environment. 
+
+The Design System team is not responsible for these resources and is unable to provide support for them. If you have support or feature requests please contact the owner.
+
+
+## Create flow diagrams
+
+[GOV.UK Design System Flow Diagrams](https://github.com/dashouse/govuk-design-system-flow-diagrams)⁠ -  
+A Sketch file for creating flow diagrams of GOV.UK services
+
+[GOV Flow](https://github.com/charlesrt/gov-flow)⁠ -  
+A Sketch file for creating flow diagrams of GOV.UK services
+
+[GOV Flow for Google Drawings](https://github.com/charlesrt/gov-flow)⁠ -  
+Flow diagram templates for Google drawings, based on GOV Flow
+
+https://www.beatnic.co.uk/2019/10/04/google-drawing-template-gov-flow-kit/
```

Comment:
Looks like this one needs updating to proper Markdown link style, and have a description added.